.github/workflows/docker-latest.yml

```
@@ -0,0 +1,115 @@
+        with:
+          images: ghcr.io/${{ github.repository }}
+          tags: latest-${{ env.ARCH }}
+
+      - name: Build and push by digest
+        id: build
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          target: release
+          push: true
+          platforms: ${{ matrix.platform }}
```

actionlint (1.7.4)

property "platform" is not defined in object type {build: {platform: string; runner: string}}

(expression)